.github/workflows/on_prerelease.yaml

```diff
@@ -8,128 +7,34 @@ on:
       - 'v*'
 
 env:
```

Comment:
I would move these env out of the global and to the specific jobs when they need it which I think is the extra test

Reply:
sure, done!